test/integration/developer_portal/api_docs/account_data_controller_test.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class DeveloperPortal::ApiDocs::AccountDataControllerTest < ActionDispatch::IntegrationTest
+  include DeveloperPortal::Engine.routes.url_helpers
+
+  def setup
+    @provider = FactoryBot.create(:provider_account)
+    @buyer = FactoryBot.create(:simple_buyer, provider_account: provider)
```

Comment:
Maybe we could use `buyer_account` here. That will make the admin user to be activated by default and thus we ensure the differences between the two test cases below are due to missing login only and not missing activation. WDYT?

Reply:
right, nice discovery 👍 💃